x/payment/keeper/keeper.go

```diff
@@ -34,6 +34,11 @@ func (k Keeper) Logger(ctx sdk.Context) log.Logger {
 
 //  MsgPayForMessage moves a user's coins to the module address and burns them.
 func (k Keeper) PayForMessage(goCtx context.Context, msg *types.MsgPayForMessage) (*types.MsgPayForMessageResponse, error) {
+	ctx := sdk.UnwrapSDKContext(goCtx)
+	ctx.EventManager().EmitEvent(
+		types.NewPayforMessageEvent(sdk.AccAddress(msg.Signer), msg.GetMessageSize()),
```

Comment:
@evan-forbes is this the right information we want to emit? We emit the signer address and the message size

Reply:
yes, that's perfect imho. If we need to add anything, we still can later.

Reply:
converting directly from a string to an `sdk.AccAddress` will result in some things weird here. Instead, we should probably change `NewPayForMessageEvent` to accept a string instead, as that's what we are converting to anyway in that function.